Framework/PythonInterface/mantid/plots/plotfunctions.py

```diff
@@ -110,74 +134,103 @@ def plot(workspaces, spectrum_nums=None, wksp_indices=None, errors=False,
     :param log_values: An optional list of log values to plot against.
     :return: The figure containing the plots
     """
-    plot_font = ConfigService.getString('plots.font')
-    if plot_font:
-        if len(mpl.rcParams['font.family']) > 1:
-            mpl.rcParams['font.family'][0] = plot_font
+    # Select the MatrixWorkspaces
```

Comment:
I was suprised to see so many changes required in the standard `plot` here. Could you help me understand the requirement?

Reply:
My original plan is to make plot() to handle both MatrixWorkspace and MDHistoWorkspace.  Here is how the change was from.  I later decided to implement a parallel set of methods for MDHistoWorkspace.  So method plot is reset of original status.